std_msgs/Header header
uint8[59] received_data
uint8[5]  get_data_com
uint8[20]  gimbal_control
